zookeeper-api/src/main/java/org/apache/helix/zookeeper/util/HttpRoutingDataReader.java

```diff
@@ -102,7 +102,8 @@ public static MetadataStoreRoutingData getMetadataStoreRoutingData()
    * @throws IOException
    */
   private static String getAllRoutingData() throws IOException {
-    // Note that MSDS_ENDPOINT should provide high-availability - it risks becoming a single point of failure if it's backed by a single IP address/host
+    // Note that MSDS_ENDPOINT should provide high-availability - it risks becoming a single point
+    // of failure if it's backed by a single IP address/host
```

Comment:
A little of topic. Do we have a issue to track this one? If we need to enhance this part, this is the number one issue in my opinion.

Reply:
We do not - could you please create an issue for it? Not an immediate blocker.